### ISSUE TITLE
Grid for layout

### ISSUE THREAD

**Issue by Tevyn** [WTW-IM/es-components#462]
To speed up layout of UIs and help with UI consistency, we should add a grid component. 

Bibo and team have had success pulling in material's grid and changing it's specs, so that might be an easy-ish way for a quick implementation. 

For the gutter and column number, sizing, and customization of the grid, we should probably stick to default bootstrap numbers found here: https://getbootstrap.com/docs/4.0/layout/grid/#grid-options
TL;DR: 12 columns, 15px gutter on each side of a column.

**Comment by stevematney** [comment]
We may be able to do something a little more lightweight than adapting Bootstrap by utilizing a library like [`react-grid-system`](https://www.npmjs.com/package/react-grid-system) (or something else if someone knows of a better one out there. This way we get an ad-hoc grid system without the extra weight of a Bootstrap dependency.

**Comment by bradyclifford** [comment]
Instead of bringing in a grid system, what if we just added to ES Components either a `Grid` container or standards into our `viaTheme` so we have a goto for the column and gutter sizes as @Tevyn  mentioned.

**Comment by zlaja1983** [comment]
As you might have heard Clifford, Brady (Salt Lake City) our designer Paul is putting something together that will go over of all the available grid systems and present pros/cons of using each one. One thing about the bootstrap, is that you would not have to pull in the whole library but rather a specific module, for example it'd be grid layout CSS. This way it'd give us only a subset of functionality of bootstrap and we would not pull in Components for example. Also their 4.x grid layout is based on mobile first approach and is utilizing flexbox underneath. It'd be very trivial to integrate 
[grids_compare.xlsx](https://github.com/WTW-IM/es-components/files/5561765/grids_compare.xlsx)
 using https://github.com/twbs/bootstrap-npm-starter.

**Comment by Darrken** [comment]
I like the idea of choosing an existing CSS class-based grid and using it as-is rather than trying to componentize it. That should suit us fine and we wouldn't have to tie it to a specific framework like React. As mentioned above, Paul is looking into several to see what might be the easiest to adopt, and if we can agree on one we can create a sample page with the recommended gutter/column/etc. settings.

**Comment by Darrken** [comment]
Addressed in #506
